src/pybitmessage

```diff
@@ -8,4 +8,7 @@ script_file = os.path.join(dist.location, dist.key, 'bitmessagemain.py')
 new_globals = globals()
 new_globals.update(__file__=script_file)
 
-execfile(script_file, new_globals)
```

Comment:
This is probably ok, but other parts of the code refuse to run in python3, or may be not compatible. If you remove the refusals, you'll get more errors and those need to be fixed. However, maybe it makes sense to fix those first before allowing this script to run in python3 as that may be a lot of work.